package.json

```diff
@@ -1,92 +1,100 @@
 {
```

Comment:
Please try to avoid reformatting files. At some point we'll enable `eslint` on the repository and have an enforced style everywhere, but we're not doing that today.